package.json

```diff
@@ -23,8 +23,7 @@
     "chokidar": "^1.7.0"
   },
   "devDependencies": {
-    "eslint": "^4.4.1",
```

Comment:
> note, however, that Yarn doesn't install executables (`.bin/eslint`) for transitive dependencies

Doesn't this make this a bad idea?

Reply:
To be honest, I'm not too interested in adding (and thus maintaining) special casing for non-standard tooling - think Turbo. IMO both README and `package.json` should just assume npm, as that's always available.

Of course anyone's free to use Yarn, Turbo or whatever, and I applaud their initiative. But if they do, they should be capable of figuring out how to fill the gaps.

Reply:
Good point. Ok 👍